tests/compiler/std/string-casemapping.optimized.wat

```diff
@@ -2548,6 +2548,8 @@
              i32.eq
              local.get $3
              i32.const 2
+             i32.sub
+             i32.const 0
```

Comment:
This is regression. It seems need revert some simplified arm in ZeroRemover in binaryen